app/views/participant_signup/_contact.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= f.inputs :name => "Contact Information" do -%>
```

Comment:
One more thought. I have removed questions from the form about asking for "Bad address", "Bad phone information" and "Bad email" (https://github.com/uwdreamproject/DreamSIS/blob/master/app/views/participants/fields/_contact.html.erb#L5). It'll be tedious to enclose that in an if-else. What do you think about this particular case where I have to remove specific questions from the partials which would be non-sensical to ask to an actual participant?